src/Integration.Vsix/Analysis/AnalysisStrings.resx

```diff
@@ -170,16 +170,16 @@
     <value>SonarLint: Analyzing {0}</value>
   </data>
   <data name="MSG_AnalysisComplete" xml:space="preserve">
-    <value>Finished analyzing {0}, analysis time: {1}s</value>
+    <value>Finished analyzing {0} with id {1}, analysis time: {2}s</value>
```

Comment:
The analysisId is Nullable so there is a chance that the message will be `...with id null...` which feels a bit weird

Reply:
That's just for legacy analyzers. It would be too much work doing separate strings for the 2 cases